docs/source/user_guides/basic_conceptions.rst

```diff
@@ -139,165 +148,185 @@ That's what the agent consists of:
   no transition for the message given by user is found.
   It transfers user to ``greeting_node`` no matter what user writes.
 
-* ``pipeline`` is a special object that processes user requests according to provided script.
-  In order to create a pipeline, the script should be provided and two two-string tuples:
-  the first specifies initial node flow and name and the second (optional) specifies fallback
-  node flow and name (if not provided it equals to the first one by default). 
+* ``pipeline`` is a special object that traverses the script graph based on the values of user input.
+  It is also capable of executing custom actions that you want to run on every turn of the conversation.
+  The pipeline can be initialized with a script, and with labels of two nodes:
+  the entrypoint of the graph, aka the 'start node', and the fallback node
```

Comment:
Both should either have or not have quotes.

Reply:
Done